charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+data:
+  aws-ccm: |
+    defaultChartName: aws-cloud-controller-manager
+    defaultChartVersion: ""
```

Comment:
~Why is this version missing here?~ AWS CCM uses CRS to deploy, do you know why this shows up here at all?

Reply:
These show up because the code walks through the `hack/addons/kustomize/*/kustomization.yaml.tmpl` and fetches the information from there.

We could add a special case to ignore this.

Reply:
I added a case to ignore.